packages/contracts/test/test-utils/voting.ts

```diff
@@ -73,12 +73,12 @@ export async function voteWithSigners(
     votingContract.connect(signers[i]).vote(proposalId, VoteOption.Yes, false)
   );
 
-  promises.push(
+  promises.concat(
```

Comment:
`promises.concat(...)` returns a new array and doesn't change the exising array. therefore write `promises = promises.concat(...)`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat

Reply:
Fixed in 1db1130b866b84c7ebe605e2ba62ee66674ae504